include/lsst/afw/image/fits/fits_io.h

```diff
@@ -56,8 +56,14 @@ inline void fits_read_array(fits::Fits& fitsfile, ndarray::Array<PixelT, 2, 2>&
         }
         shape = shape3.last<2>();
     }
-
-    fitsfile.readMetadata(metadata, true);
+    //
+    // Note that we cannot call fitsfile.readMetadata(metadata, true)
```

Comment:
Should this note be a TODO?

Also, no need for the extra `//` comment lines above and below.